src/ontology/mondo-ingest.Makefile

```diff
@@ -181,6 +182,8 @@ config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
+all-exclusions: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
```

Comment:
I tried to do this in a nicer way, but my `make`-foo is not strong enough.
```make
ONTOLOGIES=doid ordo omim ncit icd10cm icd10who
all-exclusions:
 	$(foreach n,$(ONTOLOGIES), config/$(n)_term_exclusions.txt)
```
I tried running it but I got:
`make: config/doid_term_exclusions.txt: Permission denied`

Reply:
this is super weird, and likely a UNIX issue.. did you run `sh run.sh make config/doid_term_exclusions.txt`?

Reply:
Yeah, I ran `make all-exclusions` using `sh run.sh`.